boards.txt

```diff
@@ -37110,3 +37110,157 @@ walter.menu.EraseFlash.all=Enabled
+elecrow_crowpanel_7.menu.PartitionScheme.noota_ffat.build.partitions=noota_ffat
+elecrow_crowpanel_7.menu.PartitionScheme.noota_ffat.upload.maximum_size=2097152
+elecrow_crowpanel_7.menu.PartitionScheme.noota_3gffat=No OTA (1MB APP/3MB FATFS)
+elecrow_crowpanel_7.menu.PartitionScheme.noota_3gffat.build.partitions=noota_3gffat
+elecrow_crowpanel_7.menu.PartitionScheme.noota_3gffat.upload.maximum_size=1048576
+elecrow_crowpanel_7.menu.PartitionScheme.huge_app=Huge APP (3MB No OTA/1MB SPIFFS)
+elecrow_crowpanel_7.menu.PartitionScheme.huge_app.build.partitions=huge_app
+elecrow_crowpanel_7.menu.PartitionScheme.huge_app.upload.maximum_size=3145728
+elecrow_crowpanel_7.menu.PartitionScheme.min_spiffs=Minimal SPIFFS (1.9MB APP with OTA/190KB SPIFFS)
+elecrow_crowpanel_7.menu.PartitionScheme.min_spiffs.build.partitions=min_spiffs
+elecrow_crowpanel_7.menu.PartitionScheme.min_spiffs.upload.maximum_size=1966080
+elecrow_crowpanel_7.menu.PartitionScheme.fatflash=16M Flash (2MB APP/12.5MB FAT)
+elecrow_crowpanel_7.menu.PartitionScheme.fatflash.build.partitions=ffat
+elecrow_crowpanel_7.menu.PartitionScheme.fatflash.upload.maximum_size=2097152
+elecrow_crowpanel_7.menu.PartitionScheme.app3M_fat9M_16MB=16M Flash (3MB APP/9MB FATFS)
+elecrow_crowpanel_7.menu.PartitionScheme.app3M_fat9M_16MB.build.partitions=app3M_fat9M_16MB
+elecrow_crowpanel_7.menu.PartitionScheme.app3M_fat9M_16MB.upload.maximum_size=3145728
+
+elecrow_crowpanel_7.menu.CPUFreq.240=240MHz (WiFi/BT)
+elecrow_crowpanel_7.menu.CPUFreq.240.build.f_cpu=240000000L
+elecrow_crowpanel_7.menu.CPUFreq.160=160MHz (WiFi/BT)
+elecrow_crowpanel_7.menu.CPUFreq.160.build.f_cpu=160000000L
+elecrow_crowpanel_7.menu.CPUFreq.80=80MHz (WiFi/BT)
+elecrow_crowpanel_7.menu.CPUFreq.80.build.f_cpu=80000000L
+elecrow_crowpanel_7.menu.CPUFreq.40=40MHz (40MHz XTAL)
+elecrow_crowpanel_7.menu.CPUFreq.40.build.f_cpu=40000000L
+elecrow_crowpanel_7.menu.CPUFreq.26=26MHz (26MHz XTAL)
+elecrow_crowpanel_7.menu.CPUFreq.26.build.f_cpu=26000000L
+elecrow_crowpanel_7.menu.CPUFreq.20=20MHz (40MHz XTAL)
+elecrow_crowpanel_7.menu.CPUFreq.20.build.f_cpu=20000000L
+elecrow_crowpanel_7.menu.CPUFreq.13=13MHz (26MHz XTAL)
+elecrow_crowpanel_7.menu.CPUFreq.13.build.f_cpu=13000000L
+elecrow_crowpanel_7.menu.CPUFreq.10=10MHz (40MHz XTAL)
+elecrow_crowpanel_7.menu.CPUFreq.10.build.f_cpu=10000000L
+
+elecrow_crowpanel_7.menu.FlashMode.qio=QIO
+elecrow_crowpanel_7.menu.FlashMode.qio.build.flash_mode=dio
+elecrow_crowpanel_7.menu.FlashMode.qio.build.boot=qio
+elecrow_crowpanel_7.menu.FlashMode.dio=DIO
+elecrow_crowpanel_7.menu.FlashMode.dio.build.flash_mode=dio
+elecrow_crowpanel_7.menu.FlashMode.dio.build.boot=dio
+
+elecrow_crowpanel_7.menu.FlashFreq.80=80MHz
+elecrow_crowpanel_7.menu.FlashFreq.80.build.flash_freq=80m
+elecrow_crowpanel_7.menu.FlashFreq.40=40MHz
+elecrow_crowpanel_7.menu.FlashFreq.40.build.flash_freq=40m
+
+elecrow_crowpanel_7.menu.FlashSize.16M=4MB (32Mb)
+elecrow_crowpanel_7.menu.FlashSize.16M.build.flash_size=4MB
```

Comment:
Please rename the FlashSize option to 4M, or you can delete it completely. As the board is sold with 4MB Flash chip. 
```suggestion
elecrow_crowpanel_7.menu.FlashSize.4M=4MB (32Mb)
elecrow_crowpanel_7.menu.FlashSize.4M.build.flash_size=4MB
```

Reply:
@P-R-O-C-H-Y Thanks for the review.